src/quo2/components/notifications/activity_logs.cljs

```diff
@@ -1,5 +1,6 @@
 (ns quo2.components.notifications.activity-logs
-  (:require [quo.core :as quo]
+  (:require [clojure.string :as str]
+            [quo.core :as quo]
```

Comment:
Could we get rid of this quo1 import?

Reply:
Unfortunately no @briansztamfater, because the new quo2 `input` component is not yet implemented.

Reply:
🫡